### PR TITLE
feat: add action sheet to wizard

### LINKED ISSUE #3751: Wizard: Stack group with Action sheets is missing
#### Is this a bug, enhancement, or feature request?
Feature

#### Briefly describe your proposal.

Grouped Steps Container:
When you have multiple steps and not enough space on the screen to show all steps, you can click on the grouped container and select a step from the ActionSheet.
Example:
Screen Shot 2020-11-04 at 10 24 57 PM

### PR DESCRIPTION
**Pull request by mikerodonnell89**
#### Please provide a link to the associated issue.

fixes #3751 

#### Please provide a brief summary of this pull request.

This PR adds the action sheet to the wizard when the wizard width is small, causing the steps to stack.  Action sheet currently does not open if none of the stacked items have been previously visited - will address that in another ticket

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
No breaking changes

### CODE REVIEW

**Review by Betrozov — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

File: libs/core/src/lib/wizard/wizard.component.ts [hunk 1]
```diff
@@ -269,7 +280,9 @@ export class WizardComponent implements AfterViewInit, OnDestroy {
                 if (step.status === CURRENT_STEP_STATUS) {
                     const child = <HTMLElement>this.wrapperContainer.nativeElement.children[index];
                     this.wrapperContainer.nativeElement.scrollTo({
-                        top: child.offsetTop - this._elRef.nativeElement.querySelector('.' + WIZARD_NAVIGATION_CLASS).clientHeight,
+                        top:
```
Comment (salarenko) [inline libs/core/src/lib/wizard/wizard.component.ts:283]: Do you mind moving `this._elRef.nativeElement.querySelector('.' + WIZARD_NAVIGATION_CLASS)` to separate variable and:
``` 
this.wrapperContainer.nativeElement.scrollTo({ top: child.offsetTop - navigation.clientHeight, behavior: 'smooth' });
```

**Review by salarenko — CHANGES_REQUESTED** [review]
- [ ] When resizing the wizard, wizard step marked as selected is changing
selection-changed-resize

- [ ] No step change when clicking on step subtitle
clicking-subtitle

- [ ] Should collapsed steps look like that?
image

Docs show something like:
image

**Comment by mikerodonnell89** [comment]
> * [ ]  When resizing the wizard, wizard step marked as selected is changing
>   selection-changed-resize

@salarenko Are you sure you're not accidentally touching the mouse wheel here? That is the only way I have been able to recreate this.  Maybe the content height changing causes the div to scroll itself, those focusing the other step

**Comment by mikerodonnell89** [comment]
> * [ ]  No step change when clicking on step subtitle
>   clicking-subtitle

Can't recreate this either

**Comment by mikerodonnell89** [comment]
> * [ ]  Should collapsed steps look like that?
>   image

Are you talking about the centered text? That is coming directly from a recent change in action sheet so maybe that needs to get fixed there

**Comment by mikerodonnell89** [comment]
> Docs show something like:
> image

This does happen too, it depends on # of steps, how they have been collapsed, which one you're on etc

Screen Shot 2020-12-04 at 9 39 49 AM

**Review by salarenko — APPROVED** [review]

**Pull request merged**